### PR TITLE
Populate Python and Ruby examples

### PR DESCRIPTION
**Pull request by jirikuncar**
### What does this PR do?

Adds generated Python examples.

### Motivation

Keep all examples for generated clients in sync.

### Preview link

https://docs-staging.datadoghq.com/jirikuncar/examples/python/api/

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

**Comment by ruthnaebeck** [comment]
hello @jirikuncar - checking to see if this PR is still relevant

**Comment by jirikuncar** [comment]
Yes, it is.

**Review by ruthnaebeck — APPROVED** [review]
👍 for docs

File: config/_default/params.yaml [hunk 1]
```diff
@@ -10,32 +10,49 @@ code_languages:
...
     name: php
+    syntax: php
 
 code_language_ids:
     java: Java
     cpp: 'C++'
     php: PHP
     nodejs: NodeJS
     python: Python
+    python-beta: Python [beta]
```
Comment (jirikuncar) [inline config/_default/params.yaml:53]: @davidejones @apigirl we can modify the text for new Python and Ruby API clients here.

**Review by davidejones — APPROVED** [review]

**Pull request merged**